zapisy/apps/notifications/models.py

```diff
@@ -17,6 +17,7 @@ class NotificationPreferencesStudent(models.Model):
         "Zmiana prowadzącego grupy, do której czekasz w kolejce", default=True)
     news_has_been_added = models.BooleanField(
         "Nowa wiadomość w Aktualnościach", default=True)
+    event_decision = models.BooleanField("Decyzja odnośnie Twojego wydarzenia", default=True)
```

Comment:
```suggestion
    event_decision = models.BooleanField("Decyzja w sprawie założonego przez Ciebie wydarzenia", default=True)
```
Jak widać po innych nazwach, one nie są wcale zawsze bardzo zwięzłe, a tak jest "bardziej po polsku". Ditto dla tej samej preferencji w drugim modelu i analogicznie w treści powiadomienia.

Reply:
Ping, akurat w tym miejscu zabrakło zmiany :)

Reply:
Nie zauważyłam, że w tym pliku ten tekst występuje dwa razy... Już poprawione :)